Documentation/api.md

```diff
@@ -485,6 +485,7 @@ The `PrometheusRestrictedConfig` resource defines the settings for the Prometheu
 | Property | Type | Description |
 | -------- | ---- | ----------- |
 | scrapeInterval | string | Configures the default interval between consecutive scrapes in case the `ServiceMonitor` or `PodMonitor` resource does not specify any value. The interval must be set between 5 seconds and 5 minutes. The value can be expressed in: seconds (for example `30s`.), minutes (for example `1m`.) or a mix of minutes and seconds (for example `1m30s`.). The default value is `30s`. |
+| evaluationInterval | string | Configures the default interval between rule evaluations in case the `PrometheusRule` resource does not specify any value. The interval must be set between 5 seconds and 5 minutes. The value can be expressed in: seconds (for example `30s`.), minutes (for example `1m`.) or a mix of minutes and seconds (for example `1m30s`.). It only applies to `PrometheusRule` resources with the `openshift.io/prometheus-rule-evaluation-scope=\"leaf\"` label. The default value is `30s`. |
```

Comment:
I think
`openshift.io/prometheus-rule-evaluation-scope==\"leaf\"
should be
`openshift.io/prometheus-rule-evaluation-scope==\"leaf-prometheus\"
see from https://docs.openshift.com/container-platform/4.17/observability/monitoring/managing-alerts.html#about-creating-alerting-rules-for-user-defined-projects_managing-alerts

Reply:
correct!

Reply:
fixed